.github/ISSUE_TEMPLATE.md

```diff
@@ -1,22 +1,11 @@
 **Description:**
 
-<!-- Check first that your problem is not listed in our wiki section:
```

Comment:
This file can be purged I think, it’s the old style.... the ones in the ISSUE TEMPLATES directory are what’s actually used